### PR TITLE
modified deleted endpoint category

### PR DESCRIPTION
**Pull request by DanteIturri**
fixed issues category controller and category service

### CODE REVIEW

**Comment by DanteIturri** [comment]
resolved conflicts

File: src/main/java/com/alkemy/ong/service/CategoryService.java [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 import com.alkemy.ong.mapper.CategoryMapper;
 import com.alkemy.ong.repository.CategoryEntityRepository;
 import lombok.RequiredArgsConstructor;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
+
```
Comment (walterio123) [inline src/main/java/com/alkemy/ong/service/CategoryService.java:13]: I agree with the change

File: src/main/java/com/alkemy/ong/controller/CategoryController.java [hunk 2]
```diff
@@ -18,19 +18,23 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.List;
```
Comment (walterio123) [inline src/main/java/com/alkemy/ong/controller/CategoryController.java:19]: I agree with the change

File: src/main/java/com/alkemy/ong/controller/CategoryController.java [hunk 3]
```diff
@@ -18,19 +18,23 @@
...
+
+
+private final CategoryService categoryService;
+
 
     @GetMapping("/categories")
     public ResponseEntity <List<CategoryDto>> getAllCategories(){
         return ResponseEntity.ok(categoryService.getAll());
     }
+
```
Comment (walterio123) [inline src/main/java/com/alkemy/ong/controller/CategoryController.java:37]: I agree with the change

**Review by juan1977lopez — COMMENTED** [review]
Please, PR has conflicts, that must be resolved

**Pull request merged**